test/Tests.hs

```diff
@@ -70,7 +81,7 @@ fixtureGoldenTest fp = testGroup fp
         makeFlags :: IO ([String], Options)
         makeFlags = do
             argv <- readArgv
-            let argv' = argv ++ [name, projectfp]
+            let argv' = argv ++ [name, projectfp] ++ ["--sourcehut-source", "https://example.org"]
```

Comment:
???

Reply:
`--sourcehut-source` is required if a git remote is not found. Yeah, I should probably add that flag only if name=="sourcehut"

Reply:
done